docs/angular/api-angular/schematics/upgrade-module.md

```diff
@@ -6,7 +6,20 @@ Add an upgrade module
 
 ```bash
 ng generate upgrade-module ...
+```
+
+By default, Nx will search for `upgrade-module` in the default collection provisioned in `angular.json`.
+
+You can specify the collection explicitly as follows:
+
+```bash
+ng g @nrwl/angular:upgrade-module ...
+```
 
+Show what will be generated without writing to disk:
+
+```bash
+ng g null ... --dry-run
```

Comment:
There is still null here. This should be ng g upgrade-module or ng g @nrwl/angular:upgrade-module --dry-run